src/main/java/org/fogbowcloud/manager/core/intercomponent/xmpp/handlers/RemoteDeleteOrderRequestHandler.java

```diff
@@ -23,6 +23,8 @@ public RemoteDeleteOrderRequestHandler() {
 
     @Override
     public IQ handle(IQ iq) {
+        LOGGER.info("Received request for order: " + iq.getID());
+
```

Comment:
one possible naming convention is to have N private method here: unMarshallOrdeId, unMarshallResourceType and unMarshallFederationUser